modules/odc-discovering-and-identifying-operator-backed-bindable-services.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: PROCEDURE
+[id="odc-discovering-and-identifying-operator-backed-bindable-services_{context}"]
+== Discovering and identifying Operator-backed bindable services
+
+As a user, if you want to create a bindable service, you must know which services are bindable. Bindable services are services that can be easily consumed by applications because they expose their binding data such as credentials, connection details, volume mounts, secrets, and other binding data in a standard way. The *Developer* perspective helps you discover and identify such bindable services.
```

Comment:
```suggestion
As a user, if you want to create a bindable service, you must know which services are bindable. Bindable services are services that the applications can consume easily because they expose their binding data such as credentials, connection details, volume mounts, secrets, and other binding data in a standard way. The *Developer* perspective helps you discover and identify such bindable services.
```

Reply:
Done.

Reply:
Hello @Srivaralakshmi. I have incorporated your suggestion. Could you please confirm? Thank you!
![image](https://user-images.githubusercontent.com/89447174/179697999-9406e5e4-923b-47ee-9b70-7203ef512f3d.png)
Cc @divyanshiGupta